tests/pr_test.go

```diff
@@ -45,22 +44,22 @@ func setupOptions(t *testing.T, prefix string, terraformDir string) *testhelper.
 	return options
 }
 
-// func TestRunStandardExample(t *testing.T) {
-// 	t.Parallel()
+func TestRunStandardExample(t *testing.T) {
+	t.Parallel()
 
-// 	options := setupOptions(t, "base-ocp", standardExampleTerraformDir)
+	options := setupOptions(t, "base-ocp", standardExampleTerraformDir)
 
-// 	output, err := options.RunTestConsistency()
+	output, err := options.RunTestConsistency()
 
-// 	assert.Nil(t, err, "This should not have errored")
-// 	assert.NotNil(t, output, "Expected some output")
-// }
+	assert.Nil(t, err, "This should not have errored")
+	assert.NotNil(t, output, "Expected some output")
+}
 
 func TestRunUpgradeExample(t *testing.T) {
 	t.Parallel()
 
 	// TODO: Remove this line after the first merge to primary branch is complete to enable upgrade test
-	// t.Skip("Skipping upgrade test until initial code is in primary branch")
+	t.Skip("Skipping upgrade test until initial code is in primary branch")
```

Comment:
why are you skipping this ?

Reply:
This line I will be removing in this PR itself, but once I make other changes too as I saw some failures for upgrade test locally like the below ones : 
```
Error Trace:   /go/pkg/mod/github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper@v1.5.7/testhelper/tests.go:73

/go/pkg/mod/github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper@v1.5.7/testhelper/tests.go:293
/mnt/tests/pr_test.go:67

Error:   Should be false
Test: TestRunUpgradeExample
Messages: Resource(s)   identified to be destroyed Name: kms_instance Address: ibm_resource_instance.kms_instance Actions: [delete]

tests.go:73:
Error Trace:   /go/pkg/mod/github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper@v1.5.7/testhelper

/tests.go:73
/go/pkg/mod/github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper@v1.5.7/testhelper/tests.go:293

/mnt/tests/pr_test.go:67
Error:   Should be false
Test: TestRunUpgradeExample
Messages: Resource(s)   identified to be destroyed Name: kube_key Address: ibm_kms_key.kube_key Actions: [delete]

tests.go:79:
Error Trace:   /go/pkg/mod/github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper@v1.5.7/testhelper/tests.go:79
/go/pkg/mod/github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper@v1.5.7/testhelper/tests.go:293
/mnt/tests/pr_test.go:67

Error:   Should be false
Test: TestRunUpgradeExample
Messages: Resource(s)   identified to be updated Name: cluster Address: module.ocp_base.ibm_container_vpc_cluster.cluster[0]   Actions: [update]
DIFF:
Before: {"kms_config":[{"crk_id":"***********","instance_id":"***********","private_endpoint":true}]}
After: {"kms_config":[{"private_endpoint":true}]}
```

Reply:
Upgrade checks are now added, merged as well and no error is seen.